docs/installation.md

```diff
@@ -22,6 +22,26 @@ $ nimble install owlkettle@#head
   pkg-config --libs gtk4
   ```
 
+### Properly setting up MSYS2 on Windows
+
+If you want to make sure you have the `pkg-config` in your system, 
+you need to install your MSYS2's environment toolchain.
+
+The `UCRT64` is the default environment you should use for any project, 
+but `CLANG64` is the recommended one for Nim.
+
+Choose one of them:
+
+```bash
+$ pacman --sync --refresh --sysupgrade mingw-w64-clang-x86_64-toolchain  # CLANG64
+$ pacman --sync --refresh --sysupgrade mingw-w64-ucrt-x86_64-toolchain   # UCRT64
+```
+
+Moreover there is a discussion about this topic made by 
```

Comment:
Generally when we have docs such as these we'd want to keep them as short and scannable as possible.
Imagine myself as basically your worst "customer" in that regard as I'm *notorious* for skimming.

Therefore, the authors don't matter nearly as much to me as the information. This could be done in a simple

For further information, check out [this discussion on setting up MSYS2](https://github.com/can-lehmann/owlkettle/discussions/144).

It might make sense to check in with can.l on this. I could see him making the (valid) argument that such a link should never be necessary, as this is already the more detailed installation help and ideally should contain everything you'd need to know.

Reply:
Having a link to the discussion for asking questions is fine as long as the main information is contained in the docs. Maybe use "Feel free to ask questions [in this discussion](...)." I agree on not mentioning the authors here. Authorship is recorded via the commits and I additionally try to credit each contributor in the release notes for new versions.

Reply:
Alright, guys! :wink:

Reply:
> Generally when we have docs such as these we'd want to keep them as short and scannable as possible.

So, you mean to use the `-Syu` command, instead?

Reply:
I wouldn't say this applies to commands. 

These are there for the most part for copy-paste purposes.
Therefore it's fine to keep those as readable as possible and they can be longer - they'll only get read if somebody wants to understand this a bit more in detail.
If they don't folks will just copy-paste-execute-done.

Reply:
Fixed in [discussions should only be used for discussions and questions](https://github.com/can-lehmann/owlkettle/pull/146/commits/db05c7a61b5535b6f8edd6c34c9e196733074ce5) and [use a shorter flags for pacman commands](https://github.com/can-lehmann/owlkettle/pull/146/commits/42fadc70de7dd1c54d6ebd276f00e0766bc45fad)

Reply:
Alrighty :smile: